.github/workflows/generate-theme-doc.yml

```diff
@@ -6,6 +6,21 @@
     paths:
       - "themes/index.js"
 
+permissions:
+  actions: read
+  checks: read
+  contents: write
```

Comment:
@anuraghazra, can you restrict the permissions of the `GITHUB_TOKEN`?

Reply:
@rickstaa This alerts can be dismissed with "Used only in tests" reason. We cannot resolve them because this permissions required for workflows running. I think that if you approve it we can merge it now.

Reply:
Not sure whats this about, but approved.

Reply:
> @rickstaa This alerts can be dismissed with "Used only in tests" reason. We cannot resolve them because this permissions required for workflows running. I think that if you approve it we can merge it now.

Thanks for the explanation, makes sense!